datafusion/expr/src/planner.rs

```diff
@@ -114,8 +114,12 @@ pub trait UserDefinedSQLPlanner {
 
 /// An operator with two arguments to plan
 ///
-/// Note `left` and `right` are DataFusion [`Expr`]s but the `op` is the SQL AST operator.
-/// This structure is used by [`UserDefinedSQLPlanner`] to plan operators with custom expressions.
+/// Note `left` and `right` are DataFusion [`Expr`]s but the `op` is the SQL AST
+/// operator.
+///
+/// This structure is used by [`UserDefinedSQLPlanner`] to plan operators with
+/// custom expressions.
+#[derive(Debug, Clone)]
```

Comment:
I think we don't need Clone 🤔  Same as `RawFieldAccessExpr ` and `PlannerResult `. Others looks good to me

Reply:
I agree there isn't a critical usecase for it now, but I figured it didn't hurt. If you feel strongly I will remove `Clone`

Reply:
I think it is not a critical issue.